### PR TITLE
Add block audit faq

### PR DESCRIPTION
**Pull request by hunicus**
These changes add a FAQ on block audits that describes what they are and how they work. There are also small updates to the existing block health FAQ.

They should cover topics raised by various users over the past few days:
- what an expected block is and how it is determined
- what a block audit shows
- what block health means
- how the block health score can be so high when the actual block is so different from the expected block
- what the colors in the block audit view mean (#3026)

Separately, I propose that we consider renaming "Projected Block" to "Expected Block" on the block page...the idea being that the next block is "projected" until the moment a block is mined, at which point the snapshot of the next projected block becomes the "expected" block for that particular block height. Might seem like splitting hairs on wording but wiz and I discussed that "expected vs actual" is more idiomatic than something like "projected vs actual".

## Block Audit

Screenshot from 2023-02-13 03-30-38

## Block Health

Screenshot from 2023-02-13 06-31-57

### CODE REVIEW

File: frontend/src/app/docs/api-docs/api-docs.component.html [hunk 1]
```diff
@@ -218,17 +218,41 @@ <h3 *ngIf="( item.type === 'category' ) && ( item.showConditions.indexOf(network
...
+  <p>The number of transactions appearing in both <span class='math'>s<sub>expected</sub></span> and <span class='math'>s<sub>actual</sub></span> is used (instead of a block's full transaction count) in order to minimize chances that block health is inadvertently impacted by transactions that were most likely not intentionally excluded:</p>
   <ul>
     <li>recently-broadcast transactions, since the miner may simply not have received them</li>
     <li>certain low-feerate transactions, since the miner may have opted to replace them with more profitable out-of-band transactions</li>
   </ul>
-  <p>Mempool uses a re-implementation of Bitcoin Core's transaction selection algorithm to determine the transactions it expects to see in the next block.</p>
+  <p>As a result, block health is <i>not</i> intended to be a measure of how closely an expected block resembles an actual block. The actual block can be vastly different from the expected block, but if no transactions appear to be intentionally excluded, it will have a high health rating (<a [routerLink]="['/block/0000000000000000000515e202c8ae73c8155fc472422d7593af87aa74f2cf3d']">extreme example</a>).</p>
```
Comment (mononaut) [inline frontend/src/app/docs/api-docs/api-docs.component.html:254]: > As a result, block health is <i>not</i> intended to be a measure of how closely an expected block resembles an actual block.

this sentence is repeated verbatim in line 245, I'm not sure if that's unintentional or for deliberate emphasis?

Comment (hunicus) [reply frontend/src/app/docs/api-docs/api-docs.component.html:253]: Nope, I just forgot to remove that line. Thanks for catching that!

**Comment by hunicus** [comment]
Added a note about block audit and block health only being supported on official instances. The FAQs themselves also only show on official instances.

Screenshot from 2023-02-13 19-35-24

**Review by softsimon — APPROVED** [review]
tested ACK @ v2.5.0-dev [[7aca0450](https://github.com/mempool/mempool/commit/7aca0450)]

**Pull request merged**